app/services/data/get-court-reporters-id-range.js

```diff
@@ -0,0 +1,9 @@
+const knex = require('../../../knex').appSchema
```

Comment:
If you use `const knex = require('../../../knex').stagingSchema` for this, you won't have to explicitly set it below.